docs/source/deployment/lambda.mdx

```diff
@@ -394,3 +497,38 @@ exports.handler = serverlessExpress({ app });
 ```
 
 </MultiCodeBlock>
+
+
+## Customizing HTTP behavior
```

Comment:
Isn't this what middleware allows for now?

I'm not opposed to keeping the vendia stuff in our docs / on this page, but maybe more as a footnote here under a differently named section?

Reply:
I think there is still value in noting the express integration given that our implementation assumes (without event interception) that the `ApolloServer` is the _only_ thing running on the handler. If a true proxy integration is desired, with multiple route-based handlers, the vendia implementation should be noted.

I think I updated it to this wording elsewhere, but I think "Customizing HTTP Routing Behavior" might be a better fit.